tchannel/sync/client.py

```diff
@@ -103,7 +103,6 @@ def __init__(
 
         )
         self._threadloop = threadloop or ThreadLoop()
```

Comment:
if the caller passes an externally created threadloop, maybe tchannel should not be trying to start it at all.